os/android/libhwcservice/utils/mode.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include <binder/IServiceManager.h>
+#include <utils/String8.h>
```

Comment:
The executable makes use of HWC service to test the functionalities and it doesn't call API directly defined in HWC shared lib.  That case should I still include platformdefines.h ?. 
Please correct me If you have some other thoughts.